Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:3.11
+
+ENV HOME=/opt/build
+
+RUN apk add               \
+  --no-cache              \
+  --                      \
+  alpine-sdk              \
+  apk-tools               \
+  && addgroup root abuild
```

Comment:
Is this command related to the `apk add` in such a way that it needs to be in an atomic layer?

Or is it more likely that the rate of change of this command will be *lower* than the `apk add` line? Meaning it could come first so it would make better use of the docker layer cache?

Reply:
These days I tend to group all adjacent `RUN` statements. 
The `apk add` is heavy in terms of runtime, and the `addgroup` is very fast. So splitting the adgroup would create an additional layer. And each layer adds another ~0.5s overhead that is noticable. So adding a layer is itself a heavy operation. Given I predict a very low change rate for the `addgroup` I think its best to have it in that layer.